h2o-py/h2o/h2o.py

```diff
@@ -904,7 +904,7 @@ def log_and_echo(message=""):
     api("POST /3/LogAndEcho", data={"message": str(message)})
 
 
-def remove(x):
+def remove(x, cascade=True):
```

Comment:
@michalkurka one question: for backwards compatibility. Should I default `cascade=False` (although I don't like it as original contract was for `remove` to delete object+dependencies).
Or should we document that users should use `cascade=False` if they have any issue to obtain the previous behaviour.